app/models/concerns/orchestration/ssh_provision.rb

```diff
@@ -99,6 +99,6 @@ def validate_ssh_provisioning
 
   def provision_host
     # usually cloud compute resources provide IPs but virtualization do not
-    provision_interface.ip6 || provision_interface.ip || provision_interface.fqdn
```

Comment:
You can use `.ip6.presence` to force an empty string to nil and keep `||`

Reply:
changed, thanks!